Loop/Views/CarbEntryView.swift

```diff
@@ -131,6 +131,22 @@ struct CarbEntryView: View, HorizontalSizeClassOverride {
 
             CardSectionDivider()
             
+            if let source = viewModel.carbsSource {
+                HStack(spacing: 2) {
+                    Text("Source")
+                        .foregroundColor(.primary)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                    
+                    Spacer()
+                    
+                    Text(source.name)
+                        .foregroundColor(Color(.secondaryLabel))
+                }
+                .accessibilityElement(children: .combine)
+                
+                CardSectionDivider()
+            }
+            
```

Comment:
This is a way I thought we could display the source in the CarbEntryView if it's available:

![Simulator Screenshot - iPhone 16 Pro - 2025-04-22 at 14 44 12](https://github.com/user-attachments/assets/3772b382-e37a-4a3f-b5f2-d61ebc35e8f3)

Reply:
We could also make editing disabled if presented via deeplink

Reply:
I'm not sure we'd want to disable editing?  Seems like that should always be a good thing, for cases where the originating app might not be correct? Or the user changes their mind?